frontend/public/components/masthead.tsx

```diff
@@ -9,12 +10,96 @@ import * as dedicatedLogoImg from '../imgs/openshift-dedicated-logo.svg';
 import { FLAGS, connectToFlags, flagPending } from '../features';
 import { authSvc } from '../module/auth';
 import { Dropdown, ActionsMenu } from './utils';
+import { SoftwareDetails } from './software-details';
 
 import { coFetchJSON } from '../co-fetch';
 import { SafetyFirst } from './safety-first';
 
 const developerConsoleURL = (window as any).SERVER_FLAGS.developerConsoleURL;
 
+export class HelpMenu extends React.Component<HelpMenuProps, HelpMenuState> {
+  constructor (props) {
+    super(props);
+    this.state = { showModal: false };
+    this.close = this.close.bind(this);
+    this.open = this.open.bind(this);
+    this.redirect = this.redirect.bind(this);
+  }
+
+  close() {
+    this.setState({ showModal: false });
+  }
+
+  open() {
+    this.setState({ showModal: true });
+  }
+
+  redirect() {
+    window.open('https://docs.openshift.com/online/welcome/index.html', '_blank');
+  }
+
+  render() {
+    const {username} = this.props;
+    return <React.Fragment>
+      <ActionsMenu
+        actions={[
+          {label: 'Documentation', callback: this.redirect},
+          {label: 'About', callback: this.open}]}
+        noButton
+        noCaret
```

Comment:
Do we want to keep the caret? We had it in the previous console:

<img width="309" alt="screen shot 2018-10-31 at 11 46 01 am" src="https://user-images.githubusercontent.com/1167259/47800501-ca63a080-dd02-11e8-9ab7-95fde406aab6.png">

Reply:
@serenamarie125 and @bmignano, do you want the caret?

Reply:
[This doc](https://www.patternfly.org/pattern-library/application-framework/masthead/#design) says no caret